src/integrations/action-handlers/actionkit-rsvp.js

```diff
@@ -13,29 +14,36 @@ export const instructions = () =>
   `;
 
 export async function available(organizationId) {
```

Comment:
I'm pro the switch to getting passed `organization` (e.g. here https://github.com/MoveOnOrg/Spoke/pull/1524/files#diff-d77f85a94d07a5fa7335c65464c11578R62) -- but I think that means we need to (or should) switch this to take `(organization, user)` -- benefit is also that we don't need to do the lookup below.
Since this is our moment to do something backwards incompatible with this api, let's do that part too.

Reply:
Ack.  @schuyler1d what do you think about closing this PR and moving forward with a new one that includes all this plus the remainder of the work on the action-handler framework?